gap/oper.gi

```diff
@@ -1005,13 +1005,13 @@ function(digraph, edge)
 end);
 
 InstallMethod(IsDigraphEdge, "for a digraph, pos int, pos int",
```

Comment:
Change the string please to agree with the args

Reply:
Sorry,